packages/babel-traverse/src/path/family.js

```diff
@@ -17,6 +17,38 @@ function addCompletionRecords(path, paths) {
   return paths;
 }
 
+function completionRecordForSwitch(cases, paths) {
+  for (let i = 0, caseLen = cases.length; i < caseLen; i++) {
```

Comment:
is there any reason to cache `caseLen` here?

Reply:
I would prefer to cache the array length always. Your views please.

Reply:
This should be a `for (const case of cases) {}` loop. You make no use of the loop variables other than `cases[i]`.

Reply:
I changed to `for of`